spec/acceptance/nodesets/docker/centos-6.yml

```diff
@@ -1,15 +1,23 @@
-# This file comes from puppetlabs-apache, and is released under the apache-2.0 licence
-# https://github.com/puppetlabs/modulesync_configs/blob/master/moduleroot/spec/acceptance/nodesets/docker/centos-7.yml
+---
+# This file is managed via modulesync
+# https://github.com/voxpupuli/modulesync
+# https://github.com/voxpupuli/modulesync_config
 HOSTS:
   centos-6-x64:
+    default_apply_opts:
+      order: random
+      strict_variables:
     platform: el-6-x86_64
-    hypervisor: docker
+    hypervisor : docker
     image: centos:6
     docker_preserve_image: true
     docker_cmd: '["/sbin/init"]'
-    # install various tools required to get the image up to usable levels
     docker_image_commands:
-      - 'yum install -y crontabs tar wget openssl sysvinit-tools iproute which initscripts'
+      - 'rm -rf /var/run/network/*'
+      - 'yum install -y crontabs tar wget'
```

Comment:
we shoukd probably update the nodeset in modulesync to also contain the other packages (of they are needed)